contrib/dlb/source/BUILD

```diff
@@ -26,9 +26,10 @@ make(
 
 envoy_cc_contrib_extension(
     name = "connection_balancer",
-    srcs = [
-        "connection_balancer_impl.cc",
-    ],
+    srcs = select({
+        "//bazel:linux": ["connection_balancer_impl.cc"],
```

Comment:
Shouldn't this be `linux_x86_64` for consistency with the rest of the BUILD file (and also the feature is x86 only)?

Reply:
yes, thanks for point out.